README.md

```diff
@@ -78,6 +78,15 @@ Currently you must use a version of TensorFlow from
 instructions](#build-the-tensorflow-backend-with-custom-tensorflow)
 below.
 
+### How does the TensorFlow backend manage GPU memory?
+
+The TensorFlow backend does not "release" GPU memory until the Triton process
+exits. TensorFlow uses a pool allocator and so it retains any memory it
```

Comment:
It will be a good idea to include a link to this TF issue. This behavior might change in the future with any enhancements in TensorFlow. Having a link here will help us in detecting when this documentation gets obsolete.